eppo/src/main/java/cloud/eppo/android/dto/Allocation.java

```diff
@@ -1,25 +1,68 @@
 package cloud.eppo.android.dto;
 
+import java.util.Date;
 import java.util.List;
+import java.util.Set;
 
 public class Allocation {
-    private float percentExposure;
 
-    private List<Variation> variations;
+    private String key;
 
-    public float getPercentExposure() {
-        return percentExposure;
+    private Set<TargetingRule> rules;
+
+    private Date startAt;
+
+    private Date endAt;
+
+    private List<Split> splits;
+
+    private boolean doLog;
+
+    public String getKey() {
+        return key;
+    }
+
+    public void setKey(String key) {
+        this.key = key;
+    }
```

Comment:
Are constructor parameters not supported in this version of Java? Lots of getters and setters 😮